### PR TITLE
Add validation for uninitialized fields in LinuxOperatingSystem

### PR DESCRIPTION
**Pull request by YoshiEnVerde**
Added validation on use of fields that might be zero after initialization error for the LinuxOperatingSystem class

### CODE REVIEW

**Comment by dbwiddis** [comment]
Looks good to me!

File: oshi-core/src/main/java/oshi/software/os/linux/LinuxOperatingSystem.java [hunk 1]
```diff
@@ -247,9 +247,13 @@ public OSProcess getProcess(int pid) {
...
         proc.setVirtualSize(ParseUtil.parseLongOrDefault(split[22], 0L));
         proc.setResidentSetSize(ParseUtil.parseLongOrDefault(split[23], 0L) * this.memoryPageSize);
-        proc.setKernelTime(ParseUtil.parseLongOrDefault(split[14], 0L) * 1000L / hz);
-        proc.setUserTime(ParseUtil.parseLongOrDefault(split[13], 0L) * 1000L / hz);
-        proc.setStartTime(bootTime + ParseUtil.parseLongOrDefault(split[21], 0L) * 1000L / hz);
+        if(hz != 0) {
+            proc.setKernelTime(ParseUtil.parseLongOrDefault(split[14], 0L) * 1000L / hz);
+            proc.setUserTime(ParseUtil.parseLongOrDefault(split[13], 0L) * 1000L / hz);
+            proc.setStartTime(bootTime + ParseUtil.parseLongOrDefault(split[21], 0L) * 1000L / hz);
+        } else {
```
Comment (dbwiddis) [inline oshi-core/src/main/java/oshi/software/os/linux/LinuxOperatingSystem.java]: Actually, on second thought.... if hz == 0 we should explicitly re-set it to its default value of 1000L.

Comment (YoshiEnVerde) [reply oshi-core/src/main/java/oshi/software/os/linux/LinuxOperatingSystem.java]: Easily done! PR updated!

**Comment by dbwiddis** [comment]
Fixes #413

**Pull request merged**